examples/cpp/eigen_opencv/README.md

```diff
@@ -17,3 +15,58 @@ You can find the example at <https://github.com/rerun-io/cpp-example-opencv-eige
   <source media="(max-width: 1024px)" srcset="https://static.rerun.io/cpp-example-opencv-eigen/2fc6355fd87fbb4d07cda384ee8805edb68b5e01/1024w.png">
   <source media="(max-width: 1200px)" srcset="https://static.rerun.io/cpp-example-opencv-eigen/2fc6355fd87fbb4d07cda384ee8805edb68b5e01/1200w.png">
 </picture>
+
+This is a minimal CMake project that shows how to use Rerun in your code in conjunction with [Eigen](https://eigen.tuxfamily.org/) and [OpenCV](https://opencv.org/).
+
+
+# Used Rerun Types
```

Comment:
We've recently decided to not use Title Casing for markdown headers. So we only Capitalize names, like `Rerun`
```suggestion
# Used Rerun types
```

Reply:
All the examples should change then. Should I make it in that batch and later with a different pull request change all the other ones?

Reply:
I can batch change it after merge